### PR TITLE
IP ranges and Log level

### PR DESCRIPTION
**Pull request by biorus**
When there're more than one IP rules (IP ranges or IPs), the user is not able to sign in because its IP does not pass one the them. For example, 159.15.129.0/24 and 159.15.234.0/24. I changed the condition in order to allow access when one rule is passed.

I also added Log4j files and category to enable Log level by default.

### CODE REVIEW

File: src/main/java/ru/emdev/security/auth/util/PreferencesUtil.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+				PropsKeys.EXTENDED_AUTH_ENABLED_BY_COMPANY, false);
+	}
+
+	public static boolean isEnabledOverrideUserSettings(long companyId) {
+
+		return getBoolean(companyId,
+				PropsKeys.EXTENDED_AUTH_ENABLED_OVERRIDE_USER_SETTINGS, false);
+	}
+
+	public static boolean isEnabledAccessAdmin(long companyId, long userId) {
```
Comment (akakunin) [inline src/main/java/ru/emdev/security/auth/util/PreferencesUtil.java:34]: Do I understand this feature correctly: if this property set into true - then extended checking is avoided for admin users?

Comment (biorus) [inline src/main/java/ru/emdev/security/auth/util/PreferencesUtil.java:34]: Yes, It is so the admins users will have access.

**Comment by akakunin** [comment]
Hi! Thank you for pull-request. Could you contact me by email: akakunin at emdev.ru to discuss organizational questions?

**Comment by biorus** [comment]
Hi. You're welcome. I sent an email to that email.

**Pull request merged**